java-logging-appinsights/README.md

```diff
@@ -66,38 +66,47 @@ Sample of `AI-Agent.xml`:
 This file must be placed in a `<repository-root>/lib` directory for applications running on CNP along with the agent jar that matches the current app insights version here
 Retrieve the jar from github, i.e. https://github.com/Microsoft/ApplicationInsights-Java/releases
 
-For custom telemetry metrics implement `AbstractAppInsights` already provided within module. It contains telemetry client ready for usage.
+`AbstractAppInsights` is now `deprecated` TelemetryClient can be `autowired`  to implement custom telemetry metrics.
 
 ### Configuration defaults
 
-#### Modules
+#### Modules configured by spring starter
 
 - WebRequestTrackingTelemetryModule
 - WebSessionTrackingTelemetryModule
 - WebUserTrackingTelemetryModule
 
-#### Initialisers
+#### Initializers configured by spring starter
 
 
-##### Context
+###### Context
 
 - DeviceInfoContextInitializer
 - SdkVersionContextInitializer
 
-##### Telemetry
+###### Telemetry
 
-- SequencePropertyInitializer
-- TimestampPropertyInitializer
 - WebOperationIdTelemetryInitializer
 - WebOperationNameTelemetryInitializer
 - WebSessionTelemetryInitializer
 - WebSyntheticRequestTelemetryInitializer
 - WebUserTelemetryInitializer
 - WebUserAgentTelemetryInitializer
-- CloudRoleNameInitializer
+- CloudInfoContextInitializer
+- SpringBootTelemetryInitializer
+
+#### Initializers configured by java-logging-appinsights
+
+###### Telemetry
+
+- TimestampPropertyInitializer
+- SequencePropertyInitializer
+- WebSyntheticRequestTelemetryInitializer
+- ContextInitializer ( Custom Initializer to set component version to context)
```

Comment:
This is actually a `Context` (e.g. like above ^^ in readme) and name is different